jetty-client/src/main/java/org/eclipse/jetty/client/http/HttpClientTransportOverHTTP.java

```diff
@@ -67,4 +69,14 @@ protected HttpConnectionOverHTTP newHttpConnection(EndPoint endPoint, HttpDestin
     {
         return new HttpConnectionOverHTTP(endPoint, destination, promise);
     }
+
+    public int getHeaderCacheSize()
```

Comment:
Please annotate this method with `@ManagedAttribute("The HTTP headers cache size")`.